staging/src/k8s.io/apiextensions-apiserver/pkg/controller/openapi/controller.go

```diff
@@ -89,6 +90,30 @@ func (c *Controller) Run(staticSpec *spec.Swagger, openAPIService *handler.OpenA
 		return
 	}
 
+	// create initial spec to avoid merging once per CRD on startup
+	crds, err := c.crdLister.List(labels.Everything())
+	if err != nil && !errors.IsNotFound(err) {
```

Comment:
[crdLister.List](https://github.com/kubernetes/kubernetes/blob/df2094b3d728bd58c5f23b41add109e32fc7c301/staging/src/k8s.io/apiextensions-apiserver/pkg/client/listers/apiextensions/internalversion/customresourcedefinition.go#L47-L53) doesn't return NotFound error

nit: it could be weird if `crds` and `err` were both set, but in current implementation I think no error will be returned for `labels.Everything()`

Reply:
true